plugins/node/opentelemetry-instrumentation-aws-lambda/src/instrumentation.ts

```diff
@@ -241,6 +241,10 @@ export class AwsLambdaInstrumentation extends InstrumentationBase {
       currentProvider = currentProvider.getDelegate();
     }
 
+    if (typeof currentProvider.forceFlush === 'function') {
+      return currentProvider.forceFlush.bind(currentProvider);
+    }
+
     if (typeof currentProvider.getActiveSpanProcessor === 'function') {
```

Comment:
I have a feeling that at the time of writing, there wasn't any forceFlush on the tracer provider. Now that there is, it seems like this fallback isn't actually needed anymore (and would make the code closer to what the spec is, which is that SDK TracerProvider has forceFlush)

Reply:
Yeah, I agree. Neither "forceFlush" nor fallback property "getActiveSpanProcessor" is in the API (TraceProvider TS type) so since we depend on the SDK implementation we can simplify things.